content/disaster-recovery.md

```diff
@@ -0,0 +1,43 @@
+### 3. Externalizing the blobstore
+It would also make sense to use an external blobstore like AWS S3 which is highly available than consuming blobs from the colocated DAV blobstore. This would help to fail over to a new director with less worries of re-fetching the blobs (keeping in mind this is a time consuming process) when the need arises. As of now, it is already possible to use highly available blobstores provided by AWS, Azure, AliCloud and GCP. Switching over to a new blobstore can be smooth if there's a blobstore migration mechanism implemented using the existing BOSH Blobstore CLIs (by simply 'putting' the blobs using the s3cli from the director VM to the S3 bucket for example in case of AWS). Another way of doing this would be to recreate the blobstore externally by uploading the blobs with [--fix](https://bosh.io/docs/cli-v2/#upload-release) or running the deploy commands with [--fix-releases](https://bosh.io/docs/cli-v2/#deploy). This however is costly in terms of time required to re-fetch the blobs.
+See [this](https://bosh.io/docs/director-configure-blobstore) to start configuring the director to use an external blobstore.
+
+## Steps for recovering a bosh director in the event of an AZ outage:
+
+### 1. Isolating the director
+To isolate the director in `zone_1` from the database to prevent it from modifying the state when the zone is up, it is important that:\
+a. The database passwords via terraform are rotated\
+b. All existing database connections are killed via psql
+### 2. Deploying another director
+There might be differences in deploying BOSH directors in another AZ based on whether we use the `create-env` approach or the `bosh deploy` approach.
+
+After isolating the director deployed by `create-env` in `zone_1` the process to deploy a second one in `zone_2` can be treated similar to a scratch installation. That means that the zone used in the director's deployment manifest to is changed to `zone_2` and the `bosh-state.json` is dropped before calling [create-env](https://bosh.io/docs/cli-v2/#create-env) and setting up the second director in `zone_2`.
+
+For directors deployed by `bosh deploy`, the deployment manifest should be updated the used zone in the manifest should be set to `zone_2`. Next,the number of [instances](https://bosh.io/docs/manifest-v2/#instance-groups) in the `instance_group` has to be increased to 2 and a second static ip (available in `zone_2`) should be configured.
```

Comment:
Why the instances should be increased? Which instances? What about the static ip?

Reply:
I guess that in the case of a director (let's say B) that is deployed by another director A, A would have the original IP and instance records in it's database, and simply overwriting these records might lead to inconsistency or errors. Therefore we thought that it's probably better to configure two instances (and the other one also having another static IP).